### ISSUE TITLE
Wizard's tab doesn't get resized/realigned on side menu minimised/maximised  

### ISSUE THREAD

**Issue by suhasghodke** [creativetimofficial/material-dashboard-pro-angular#156]
Wizard's tab doesn't get resized/realigned on side menu minimised/maximised 

This is replicable on the demo site too

Steps to reproduce 
#1 : go to http://md-pro-angular.creative-tim.com/forms/wizard
#2 : fill the form and go to Address tab
#3 : click on icon on navbar to minimise the side menu

You will see the tabs doesn't get resized/realigned as per new screen size.

**Comment by GHakopian** [comment]
i fixed this in my project, add the following handler.

`
$( window ).resize( () => {
            this.deploy();
        });
public deploy = () => {
        const index = this.wizardElement.bootstrapWizard('currentIndex');
        let tabCount = this.wizardElement.find('.nav li').length;
        let  $li_width = 100/tabCount;

        let move_distance = this.wizardElement.width() / tabCount;
        let index_temp = index;
        let vertical_level = 0;

        let isMobile = $(document).width() < 600 && tabCount > 3;

        if(isMobile){
            move_distance = this.wizardElement.width() / 2;
            index_temp = index % 2;
            $li_width = 50;
            vertical_level = index / 2;
            vertical_level = vertical_level * 38;
        }

        this.wizardElement.find('.nav li').css('width',$li_width + '%');

        let step_width = move_distance;
        move_distance = move_distance * index_temp;

        let $current = index + 1;

        if($current == 1 || (isMobile && (index % 2 == 0) )){
            move_distance -= 8;
        } else if($current == tabCount || (isMobile && (index % 2 == 1))){
            move_distance += 8;
        }

        this.wizardElement.find('.moving-tab').css('width', step_width);
        $('.moving-tab').css({
            'transform':'translate3d(' + move_distance + 'px, ' + vertical_level +  'px, 0)',
            'transition': 'all 0.5s cubic-bezier(0.29, 1.42, 0.79, 1)'

        });

        $('.moving-tab').css({
            'transition': 'transform 0s'
        });
    }`

you basicly run the code in ngAfterViewInit every time the windows is resized.

**Comment by BobaFett58** [comment]
Hello, Tim!
Another one bug with this wizard on the Account page. When you open from mobile(samsungA52016), you can't unselect checkboxes.

**Comment by niesteszeck** [comment]
Hi, I made an import of the wizard that is angular2 designed, you can find it [here](https://github.com/niesteszeck/ng2-material-bootstrap-wizard)

**Comment by chelaruc** [comment]
@niesteszeck can you make it like a plugin and add it to npmjs? It will be helpful for more people.

**Comment by niesteszeck** [comment]
yes it'll. But I don't really know how yet, and for now, isn't one of my priorities. Any help from others will be welcome. 

If you want to integrate it on material dashboard pro angular, just copy the wizardform folder (I have it inside `src/app/plugins/`, import it in AppComponent and it is ready (I developed it inside mdpa). If someone want my first form using this, I can share it via email, it use a background service so it would't work directly, but is a little more complete that the one on the public repository of the plugin.

**Comment by niesteszeck** [comment]
[Here](https://github.com/niesteszeck/ng2-material-bootstrap-wizard/issues/2) is the issue, so anyone that wants to contribute is welcome.
